code/script.js

```diff
@@ -322,4 +378,6 @@ const checkMyGuess = (personToCheck) => {
 start()
 
 // All the event listeners
-restartButton.addEventListener('click', start)
+restartButton.addEventListener('click', start);
+findOut.addEventListener('click', selectQuestion);
+  //generateBoard
```

Comment:
I can not see where in the code you made the box to enter your  name (when you start)  but it was  really nice done!